docs/getting-started/installation.md

```diff
@@ -3,14 +3,31 @@ title: Installation
 ---
 
 !!! tip "Build from Source"
-    Instructions to build from source can be found [here](core-geth/developers/build-from-source/).
+    Instructions to build from source can be found [here](https://etclabscore.github.io/core-geth/developers/build-from-source/)
 
 ## Pre-built executable
 
-If you just want to download and run `geth` or any of the other tools here, this is the quickest and simplest way.
+If you just want to download and run `geth --classic` or any of the other tools here, this is the quickest and simplest way.
 
 Binary archives are published at https://github.com/etclabscore/core-geth/releases. Find the latest one for your OS, download it, (check the SHA sum), unarchive it, and run!
 
+Tip: When running Core-geth use `--classic` flag for Ethereum Classic mainnet.
+
+```shell
+$ wget https://github.com/etclabscore/core-geth/releases/download/v1.12.17/core-geth-linux-v1.12.17.zip # Update to the most current release version
+$ sudo unzip core-geth-linux-v1.12.17.zip -d /bin/ # Update to the most current release version
```

Comment:
Normally the best place for user-installed programs is in `/usr/local/bin`.
> https://unix.stackexchange.com/questions/8656/usr-bin-vs-usr-local-bin-on-linux 

```suggestion
$ sudo unzip core-geth-linux-v1.12.17.zip -d /usr/local/bin # Update to the most current release version
```

Here and other places.